app/DateTimePicker/date-picker.scss

```diff
@@ -14,61 +11,55 @@ $COLOR_LIGHT_GRAY: #e9e9ed;
     border-radius: $border-radius;
     @include themify($themes) {
         background: themed('ChartHistoryPickerBg');
-        border: 1px solid themed('ChartHistoryPickerBorder');
     }
 
-    .calendar-input {
-        background: transparent;
-        margin: 0;
-        border: none;
-        outline: none;
-        width: 100%;
-        box-sizing: border-box;
-        font-size: 1.4em;
-        font-weight: 300;
-        height: 34px;
-        padding: 6px 10px;
-        @include themify($themes) {
-            color: themed('ChartHistoryPickerTopColor');
-        }
-
-        &[readonly] {
-            cursor: default;
-        }
+    .calendar-select {
+        display: flex;
+        justify-content: center;
+        width: 120px;
+        column-gap: 4px;
```

Comment:
Please help to update `rem` to `px` all over your PR.

Reply:
Fixed

Reply:
@maryiafrantsava-deriv you need to change `px` to `rem` to make sure the sizes are responsive.

Reply:
@matin-binary what about 'em' in my PR? should I save 'em'?

Reply:
`px` to `rem` should be good enough, `em` is relative to the font-size of the element so better leave it as it is.

Reply:
Fixed